extension/chrome/src/js/storage.js

```diff
@@ -27,196 +27,247 @@ var extensionStorage = (function() {
 		checkversion: true
 	};
 
-	var cloudStorage = chrome.storage.sync;
+	let cloudStorage = chrome.storage.sync;
 	// missing api in firefox
 	if(typeof chrome === 'undefined' || typeof chrome.storage === 'undefined' || typeof chrome.storage.sync === 'undefined') {
-		var store = typeof chrome === 'undefined' && typeof browser !== 'undefined' ? browser : chrome;
+		const store = typeof chrome === 'undefined' && typeof browser !== 'undefined' ? browser : chrome;
 		cloudStorage = typeof store !== 'undefined' ? store.storage.local : typeof storage !== 'undefined' ? storage.local : null;
 		if (typeof cloudStorage === 'undefined') {
 			throw 'No storage available';
 		}
 	}
+
+	const storagePromised = {
+		set(items) {
+			return new Promise((resolve, reject) => {
+				cloudStorage.set(items, () => {
+					const err = chrome.runtime.lastError
+					if (err) {
+						reject(err)
+					} else {
+						resolve(items)
+					}
+				})
+			})
+		},
+		get(keys) {
```

Comment:
[minor] it's only used as `storagePromised.get(null)`, so possible to refactor to `get(key = null)` and `storagePromised.get()`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters seem well supported